ether-wake.c

```diff
@@ -97,10 +98,13 @@ int debug = 0;
 u_char wol_passwd[6];
 int wol_passwd_sz = 0;
 
+static int hold = 0;
+
 static int opt_no_src_addr = 0, opt_broadcast = 0;
 static int opt_nfqueue_num = -1;
 
 static int send_magic_packet();
+static int send_deferred_magic_packet();
```

Comment:
The name suggests that the function is deferring the magic packet, but that's not what's happening. Could you rename the function to `send_magic_packet_wait_online()` or something?